frontend/rolecall/src/app/cast/cast-editor-v2.component.html

```diff
@@ -1,34 +1,48 @@
 <app-loading-spinner *ngIf="!dataLoaded"></app-loading-spinner>
-<div [ngStyle]="dataLoaded ? {} : { 'display' : 'none' }" class="root-container">
+<div class="root-container"
+    [ngStyle]="dataLoaded ? {} : { 'display' : 'none' }">
   <div class="cast-selector-container">
     <div class="app-card card-margins left-card">
-      <div ngbDropdown #pieceDropdown="ngbDropdown">
-        <div class="app-card piece-select" ngbDropdownToggle>
-          <h1 class="left-card-title">
+      <div ngbDropdown
+          #pieceDropdown="ngbDropdown">
+        <div ngbDropdownToggle
+            class="app-card piece-select">
+          <h1 class="left-card-title"
+              [ngStyle]="{'font-style': selectedPiece && selectedPiece.siblingId > 0 ? 'italic' : 'normal' }">
             {{selectedPiece ? selectedPiece.name : " "}}
           </h1>
           <mat-icon class="dropdownArrow">
             arrow_drop_down
           </mat-icon>
-          <div ngbDropdownMenu aria-labelledby="Piece Dropdown" class="piece-options">
-            <button ngbDropdownItem *ngFor="let piece of allPieces" (click)="onSelectPiece(piece)">
+          <div ngbDropdownMenu
+              class="piece-options"
+              aria-labelledby="Piece Dropdown">
+            <button *ngFor="let piece of popupList"
+                ngbDropdownItem
+                (click)="onSelectPiece(piece.pieceIx)"
+                [ngStyle]="{'font-style': piece.hasSibling ? 'italic' : 'normal' }">
               {{piece.name}}
             </button>
           </div>
         </div>
       </div>
       <div class="cast-select-container">
-        <div *ngFor="let cast of filteredCasts" class="app-card cast-select"
-          [ngClass]="cast.uuid == castDragAndDrop.selectedCastUUID ? 'selected-cast' : ''"
-          (click)="setCurrentCast(cast)">
-          <h2 class="cast-name">
+        <div *ngFor="let cast of filteredCasts"
+            class="app-card cast-select"
+            [ngClass]="cast.uuid == castDragAndDrop.selectedCastUUID ? 'selected-cast' : ''"
```

Comment:
can be simplified
`[class.selected-cast]="cast.uuid === castDragAndDrop.selectedCastUUID"`

Reply:
Done.